awx/main/access.py

```diff
@@ -411,8 +415,8 @@ def check_org_host_limit(self, data, add_host_name=None):
                 )
             )
 
-        if add_host_name:
-            host_exists = Host.objects.filter(inventory__organization=org.id, name=add_host_name).exists()
+        if kwargs.get('add_host_name', None):
+            host_exists = Host.objects.filter(inventory__organization=org.id, name=kwargs.get('add_host_name')).exists()
```

Comment:
Since the previous line proves that the key is there, it's ok to do the slightly shorter `kwargs['add_host_name']`